frontend/src/lib/components/apps/editor/component/default-codes.ts

```diff
@@ -46,6 +46,18 @@ export const DEFAULT_CODES: Partial<
 export async function main(db: Resource<"postgresql"> = "$res:f/examples/demodb") {
 	const query = await pgSql(db)\`SELECT * FROM demo;\`;
 	return query.rows;
+}`
+	},
+	steppercomponent: {
```

Comment:
need the equivalent in python

Reply:
the structure is not super intuitive also need formatting
```
  // if page == 0 {
  //  if (page0Invalid) throw Error("first step invalid")
  // } else if ...
  if (final) {
     submitFullForm()
  }
```

Also we should call it `lastPage`

Reply:
Done!